Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/PipelineSwitchingTest.java

```diff
@@ -0,0 +1,41 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+import org.firstinspires.ftc.team6220_PowerPlay.GrabberCameraPipeline;
+import org.firstinspires.ftc.team6220_PowerPlay.RobotCameraPipeline;
+import org.openftc.easyopencv.OpenCvCamera;
+import org.openftc.easyopencv.OpenCvCameraFactory;
+import org.openftc.easyopencv.OpenCvCameraRotation;
+
+@Autonomous(name = "PipelineSwitchingTest")
+public class PipelineSwitchingTest extends BaseAutonomous
+{
+    @Override
+    public void runOpMode() throws InterruptedException
+    {
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1-braces
```suggestion
public class PipelineSwitchingTest extends BaseAutonomous {
    @Override
    public void runOpMode() throws InterruptedException {
```